<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="fr" lang="fr">

<head>
  <meta charset="UTF-8">
  <title>Analysis of the circadian rhythm of gene expression in the context of cancer.</title>
  <link rel="stylesheet" href="styles/style.css">
  <script src="styles/style.js"></script>
</head>

<body>
  
  <header><q> Analysis of the circadian rhythm of gene expression in the context of cancer. </q></header>

  <nav id="menu">
      <ul>
        <li><a href="#introduction">Introduction and scientific context of the project, objectives.</a></li>
        <li><a href="#work-done">Work done.</a></li>
        <li><a href="#conclusion">Conclusion, work organization, difficulties encountered, interest and prospects.</a></li>
      </ul>
  </nav>

  <main class="container">
    <section id="introduction">
      <article>
        <h2>Introduction and scientific context of the project, objectives.</h2>
        <p>A circadian rhythm is a biological rhythm lasting about 24 hours, which has at least one cycle every 24 hours
          period. It is an internal clock, nestled in the heart of the brain, which imposes the circadian rhythm on the
          body, like a conductor. All animal and plant species, molds and cyanobacteria have their own internal clock,
          set at their own pace. The most visible circadian rhythm in plants is the position of the leaves and petals,
          which straighten or open more or less depending on the time of day. In humans, the circadian rhythm is not
          innate. It begins to establish itself around the first eight weeks of life and will sit in the hypothalamus.
          The latter is made up of two suprachiasmatic nuclei each containing about 10,000 neurons which exhibit
          electrical activity oscillating over about 24 hours. This electrical activity is controlled by the cyclic
          expression of about 15 genes called
          <q><em>clock genes</em></q> .The suprachiasmatic nuclei then regulate various functions of the body through
          direct or indirect messages. They directly and indirectly innervate brain regions specializing in different
          functions.
        </p>
        <p>
          In humans, several biological processes are known to be under circadian regulation. These include the cycle
          of cell division and DNA repair, RNA processing and transport, cell metabolism, and immune system function.
          Notably, several key immune system parameters in the blood show circadian rhythms, which include the number of
          circulating hematopoietic and immunocompetent cells, as well as hormones, monoamines, and cytokines. In fact,
          communication between the circadian clock and the immune system appears to be two-way. The circadian clock
          influences the elements of the immune system in its two branches&#xA0;: the innate branch (e.g. Rhythmic expression
          of cytokines and rhythmic functions of monocytes and macrophages); and the adaptive branch (e.g. expression
          and rhythmic counting of T and B lymphocytes) of the immune system and is involved in models of immune
          diseases (e.g. Hodgkin lymphoma). Interestingly, recent data shows that the molecular clock controls metabolic
          pathways, including immunometabolism, and that the immune response could alter the rhythmicity of the clock.
        </p>
        <p>
          Physiological and behavioral mechanisms also follow a circadian rhythm in humans. Among these, we can find&#xA0;:
          the sleep / wake rhythm; variations in vigilance; body temperature; blood flow; urine and stool production;
          hair growth; potassium level.
        </p>
        <p>
          The circadian rhythm is also found in the expression of
          <q><em>clock genes</em></q>. 
          The expression of these genes in an organism changes over time. This development
          can be sudden or gradual, one-off or regular.
        </p>
        <p>
          Strictly speaking, circadian rhythms are endogenous, that is, they are generated by the body itself. These
          work even in the absence of external stimuli, under perfectly constant light and temperature conditions, for
          weeks or even months. Under constant conditions, these endogenous rhythms make it possible to maintain the 24
          hours periodicity. The period of a circadian cycle consists of three parts&#xA0;: a peak; a hollow; and a medium
          level.
        </p>
        <p>
          However, there are exogenous rhythms, caused by stimuli, which evolve independently of our biological clock
          and which do not conform to this 24 hours period. The main causes are&#xA0;: jet lag which forces the body to
          resynchronize its circadian rhythm with a new nycthemeral rhythm; nocturnal exposure to artificial light
          (especially during night work), or even to intense
          <q><em>light pollution</em></q>; a diet high in fat, which disrupts sleep by disrupting the circadian rhythm
          of adiponectin production. Indeed, concerning this last cause, adiponectin, a hormone synthesized in adipose
          tissue, plays an important role in lipid storage by regulating fat burning via an increase in insulin
          sensitivity (which promotes fat loss).
        </p>
        <p>
          Disturbances in the circadian clock are also known to be associated with several types of disease. In humans,
          several studies have confirmed that circadian desynchronization with the environment is correlated with a
          higher incidence of several types of cancer. These include non-Hodgkin lymphoma, breast, endometrial,
          prostate, and colon cancers.
        </p>
        <p>
          These informations illustrate the important regulatory role of circadian rhythm in humans and thus support
          the further development of new therapeutic approaches, involving chronotherapy and other time-dependent
          intervention strategies. As treatment, there may be mentioned, for example, phototherapy and
          chronopharmacology. Phototherapy is a therapy that relies on exposure to high intensity light for a fixed
          period of time (usually between 30 minutes and 1 hour), at a particular time which depends on the individual
          and the disorder. The patient is encouraged to reduce their exposure to light in the evening, and to remove
          any electronic device with screens from their room at least one hour before the desired bedtime.
          Chronopharmacology is the study of the effects of drugs according to circadian rhythms, in order to provide
          optimal treatment to patients and to minimize side effects. This is because drugs can have different results
          over a 24 hours period, which means that the effects of drugs are influenced by when they are administered
          during the day. Efficiency varies with circadian changes in pH, temperature, cell cycles, or protein
          synthesis.
        </p>
        <p>
          In our study, we are particularly interested in the circadian rhythm of the expression of these "clock genes"
          and their modifications in a pathological context such as the development of cancer. The project aims to study
          changes in the circadian rhythm of gene expression in humans in the context of Hodgkin lymphoma. Hodgkin
          lymphoma is a type of lymphoma (cancer of the lymphatic system) characterized by the presence of large
          atypical cells, Hodgkin cells (mononuclear) and Reed-Sternberg cells (multinuclear). These cells have been
          formally established as being of B lymphoid origin, hence the need to study them together. Our study focuses
          precisely on these two cell lines&#xA0;: the cell line of B lymphoblastoid cells on the first patient; and the
          Hodgkin lymphoma cell line on the second. Samples were taken every three hours over a 24 hours period so that
          9 time points could be analyzed for each cell line.
        </p>
        <p>
          For data processing, certain technical tools are used&#xA0;: Python and AWK which are programming languages widely
          used in bioinformatics; Galaxy, an open source web platform for data-intensive biomedical research; R which is
          an open source language and software environment for statistical and graphical calculations; DAVID which is a
          database for integrated annotation, visualization and discovery of genes; Nextcloud which is an online content
          collaboration platform.
        </p>
      </article>
    </section>
    <section id="work-done">
      <h2>Work done</h2>
      <p>
        Our circadian rhythm study is done in several stages&#xA0;: data collection, Normalization, metacycle analysis and
        finally the search for genes ontologies on David.
      </p>
      <article>
        <h3>Data collection</h3>
        <p>
          To begin our circadian rhythm study, we started by uploading the raw data of both patients from the Gene
          Expression Omnibus (GEO) data into ArrayExpress (E-MTAB-6400). ArrayExpress is one of the repositories
          recommended by major scientific journals to archive functional genomics data from microarray and sequencing
          platforms to support reproducible research. There are public databases. Next, we retrieved the configuration
          files available on the Affymetrix web page. Not all of these files are needed for normalization. For basic
          normalization, the .pgf and .clf files or only the .cdf file are required. We also retrieved the Affymetrix
          annotation file from the same page. The annotation file is a simple two-column tabular file, with the first
          column containing the names of the probe sets and the corresponding second annotations, it should not contain
          any column headers. This file is generated from the Affymetrix annotation file.
        </p>
        <p>The annotation file containing a lot of information (70,775 rows and 39 columns) as can be seen in this image&#xA0;:</p>
        <figure class="annotation"><img src="./images/annotation/img-1.png" alt="picture of annotation file">
          <figcaption>Annotation file</figcaption>
        </figure>
        <p>
          In red the two necessary columns. first column containing probe set names and second corresponding
          annotations, it should not contains any column header.
        </p>
        <p>
          To extract these two columns we used two approaches which gave the same results one with an AWK/SED script
          and another with a Python script and we got this result.
        </p>
        <figure class="colonnes"><img src="./images/annotation/img-2.png" alt="picture of column of interest within the annotation file">
          <figcaption>Column of interest within the annotation file</figcaption>
        </figure>
      </article>
      <article>
        <h3>Normalization</h3>
        <p>
          For the normalization we went to the Galaxy platform. In the first step we made the plot (graphical
          representation) of the data to visualize our non-normalized data using GIANT-QC Plot.
        </p>
        <p>
          For our two patients The GIANT-QC Plots gives us a histogram, chip analyzes at DNA (microarrays), a boxplot,
          MA plots and PCA. For reasons of space, only a few graphs will be presented&#xA0;: histogram, chip, and PCA, with
          scree plot.
        </p>
      </article>
      <article>
        <h4>The histogram&#xA0;:</h4>
        <p>
          <a href="./Data/Plot_avantNormalisation_HTML.html/Histograms1.html" target="blank"> This histogram</a> gives
          us a distribution of intensities over all the chips, each chip has a comparable range of intensity. We have
          seen that there is a single chip (HD-MY-Z_06h) which has a different curve from other chips. All other chips
          peak for the same intensity.
        </p>
      </article>
      <article>
        <h4>The DNA chips&#xA0;:</h4>
        <p>
          The DNA chip is a technique for measuring the expression of genes. Usually there are wells on a small square
          glass plate. In each well there is a piece of DNA called a probe. We incorporate radiometric target fragments
          on this plate. If the target fragment associates with the probe, then there is emission of a fluorescent
          light, the stronger the light intensity, the stronger the expression of the gene in the original sample. In
          our chip images, we do not find any artifacts that could compromise the quality of our DNA chip. On our
          results, we only see the image of the chip with luminous dots.
        </p>

        <div>
          <figure class="dna-gallery"><img src="./Data/Plot_avantNormalisation_HTML.html/Microarray_Condition10.png" alt="Microarray of condition 10"><figcaption>Microarray of LCL-HO at 06h</figcaption></figure>
					<figure class="dna-gallery"><img src="./Data/Plot_avantNormalisation_HTML.html/Microarray_Condition11.png" alt="Microarray of condition 11"><figcaption>Microarray of LCL-HO at 09h</figcaption></figure>
					<figure class="dna-gallery"><img src="./Data/Plot_avantNormalisation_HTML.html/Microarray_Condition12.png" alt="Microarray of condition 12"><figcaption>Microarray of LCL-HO at 12h</figcaption></figure>
					<figure class="dna-gallery"><img src="./Data/Plot_avantNormalisation_HTML.html/Microarray_Condition13.png" alt="Microarray of condition 13"><figcaption>Microarray of LCL-HO at 15h</figcaption></figure>
					<figure class="dna-gallery"><img src="./Data/Plot_avantNormalisation_HTML.html/Microarray_Condition14.png" alt="Microarray of condition 14"><figcaption>Microarray of LCL-HO at 18h</figcaption></figure>
					<figure class="dna-gallery"><img src="./Data/Plot_avantNormalisation_HTML.html/Microarray_Condition15.png" alt="Microarray of condition 15"><figcaption>Microarray of LCL-HO at 21h</figcaption></figure>
					<figure class="dna-gallery"><img src="./Data/Plot_avantNormalisation_HTML.html/Microarray_Condition16.png" alt="Microarray of condition 16"><figcaption>Microarray of LCL-HO at 24h</figcaption></figure>
					<figure class="dna-gallery"><img src="./Data/Plot_avantNormalisation_HTML.html/Microarray_Condition17.png" alt="Microarray of condition 17"><figcaption>Microarray of LCL-HO at 27h</figcaption></figure>
					<figure class="dna-gallery"><img src="./Data/Plot_avantNormalisation_HTML.html/Microarray_Condition18.png" alt="Microarray of condition 18"><figcaption>Microarray of LCL-HO at 30h</figcaption></figure>

					<figure class="dna-gallery"><img src="./Data/Plot_avantNormalisation_HTML.html/Microarray_Condition1.png" alt="Microarray of condition 1"><figcaption>Microarray of HD-MY-Z at 06h</figcaption></figure>
					<figure class="dna-gallery"><img src="./Data/Plot_avantNormalisation_HTML.html/Microarray_Condition2.png" alt="Microarray of condition 2"><figcaption>Microarray of HD-MY-Z at 09h</figcaption></figure>
					<figure class="dna-gallery"><img src="./Data/Plot_avantNormalisation_HTML.html/Microarray_Condition3.png" alt="Microarray of condition 3"><figcaption>Microarray of HD-MY-Z at 12h</figcaption></figure>
					<figure class="dna-gallery"><img src="./Data/Plot_avantNormalisation_HTML.html/Microarray_Condition4.png" alt="Microarray of condition 4"><figcaption>Microarray of HD-MY-Z at 15h</figcaption></figure>
					<figure class="dna-gallery"><img src="./Data/Plot_avantNormalisation_HTML.html/Microarray_Condition5.png" alt="Microarray of condition 5"><figcaption>Microarray of HD-MY-Z at 18h</figcaption></figure>
					<figure class="dna-gallery"><img src="./Data/Plot_avantNormalisation_HTML.html/Microarray_Condition6.png" alt="Microarray of condition 6"><figcaption>Microarray of HD-MY-Z at 21h</figcaption></figure>
					<figure class="dna-gallery"><img src="./Data/Plot_avantNormalisation_HTML.html/Microarray_Condition7.png" alt="Microarray of condition 7"><figcaption>Microarray of HD-MY-Z at 24h</figcaption></figure>
					<figure class="dna-gallery"><img src="./Data/Plot_avantNormalisation_HTML.html/Microarray_Condition8.png" alt="Microarray of condition 8"><figcaption>Microarray of HD-MY-Z at 27h</figcaption></figure>
					<figure class="dna-gallery"><img src="./Data/Plot_avantNormalisation_HTML.html/Microarray_Condition9.png" alt="Microarray of condition 9"><figcaption>Microarray of HD-MY-Z at 30h</figcaption></figure>
        </div>



      </article>
      <article>
        <h4>The scree plot</h4>
        <p>
          <a href="./Data/Plot_avantNormalisation_HTML.html/screePlot.html" target="blank"> The Scree plot</a> of the
          PCA allows us to know according to how many components we will find a significant separation. In this scree
          plot, we can see that the bend of the curve is at the level of the 3rd component, so we will have a difference
          according to three components in this PCA.
        </p>
      </article>
      <article>
        <h4>Principal Component Analysis (PCA):</h4>
        <p>
          It is a statistical method that makes it possible to explore so-called multivariate data (data with several
          variables). Each variable could be considered as a different dimension along which the variation of the data
          is maximum. In other words, PCA reduces the dimensions of multivariate data to two or three principal
          components, which can be visualized graphically, losing as little information as possible.
        </p>
        <p>
          <a href="./Data/Plot_avantNormalisation_HTML.html/PCA_Factor1.html" target="blank">We have a PCA</a>
          according to three components
        </p>
        <ul>
          <li>The 1st component only separates one patient from all the others, patient HD-MY-Z_06h.</li>
          <li>The 2nd component separates the different types cellular&nbsp;:
            <ul>
              <li>Negative values&#xA0;: Hodgkin Lymphoma cells</li>
              <li>Positive values&#xA0;: lymphoblastoid B cells</li>
            </ul>
          </li>
          <li>The 3rd component makes a separation but it is not very obvious.</li>
        </ul>
        <p>
          Before normalization we also looked at the Scree plot we see the number of components that are interesting to
          look at we see indeed that the 3rd component at the level of the scree plot does not allow to represent a
          significant variance of the data so it is not really interesting to look at.
        </p>
        <p>
          After having visualized our data we carried out the normalization using GIANT-Normalization with APT
          Summarize.We obtained a file containing the normalized data as a result and we then relaunched the GIANT-QC
          Plots with this data. We got the same output as the last time but with normalized data&#xA0;:
        </p>
      </article>
      <article>
        <h4>The histogram&#xA0;:</h4>
        <p>
          We can see a<a href="./Data/Plot_apresNormalisation_HTML.html/Histograms1.html" target="blank"> good
            distribution</a> of chips.
          </p>
      </article>
      <article>
        <h4>Principal Component Analysis (PCA)&#xA0;:</h4>
        <p>
          After the Normalization we see that the first component which corresponded to a technological bias due to a
          difference between the chips that has been corrected thanks to the Normalization suddenly this first component
          no longer exists because we have removed this bias which fact that the second component of the PCA without
          normalization becomes the first component.<a href="./Data/Plot_avantNormalisation_HTML.html/PCA_Factor1.html"
            target="blank"> (The new PCA)</a>
          </p>
        <p>
          We also have a file tabular containing normalized log2 transformed data as a tab-delimited matrix. The first
          line contains the <em>.CEL</em> names and the first column contains the gene names.
        </p>
        <figure class="tabular"><img src="./images/normalized_tabular_file.png" alt="A tabular file containing normalized log2">
          <figcaption>A tabular file containing normalized log2 transformed data</figcaption>
        </figure>
        <p>
          Normalization allowed to correct the bias as we saw in the non-normalized histogram there was a chip a little
          different from the others. and after normalization a good distribution of similar microarray expression was
          obtained.
        </p>
      </article>
      <article>
        <h3>Metacycle</h3>
        <p>
          The<strong> Metacycle</strong> package is mainly used for detecting rhythmic signals from large scale
          time-series data. In Metacycle there are two functions-meta2d and
          <q><em>meta3d</em></q> for detecting rhythmic signals from time-series data sets. For analyzing time-series
          data sets with individual information, meta3d is suggested, which takes use of any one of these three methods
          to analyze time-series data individual by individual and gives out integrated values based on analysis result
          of each individual. meta3d can be executed with three algorithms, JTK_cycle(JTK), ARS and LS.
        </p>
        <p>
          JTK_Cycle is a non-parametric algorithm developed. Its purpose is to identify rhythmic components in large,
          genome-scale data sets and estimate their period length, phase, and amplitude.
        </p>
        <p>
          ARSER(ARS) is a package for identifying periodic expression profiles in analyzing circadian microarray data
          and has been released under the GPL.
        </p>
        <p>
          The Lomb–Scargle(LS) periodogram approach is used to search time series of gene expression to quantify the
          periodic behavior of every gene represented on the DNA array
        </p>
        <p>
          To continue our study after normalization from Metacycle we ran meta3d with our normalized data that we put
          as a parameter as well as our design file that we created from our conditions (gene identifier), patient (1
          and 2), cell type and time as we can see&#xA0;:
        </p>
        <figure class="design"><img src="./images/design_file.png" alt="A design file">
          <figcaption>A design file as a parameter</figcaption>
        </figure>
        <p>We run meta3d with the three algorithms JTK, LS and ARS&#xA0;:</p>
          <code>
            <pre>
              library(MetaCycle)

              tmeta3d("datafile.csv", "designfile.csv",
              outdir = "metacycle_ouput", "csv",1,
              2, minper = 20, maxper = 28,cycMethodOne = "JTK",
              timeUnit = "hour", design_hrColm=4,design_dayColm = NULL,
              design_minColm = NULL,design_secColm = NULL,
              design_groupColm = NULL,design_libIDrename = NULL,
              adjustPhase = "predictedPer",combinePvalue = "fisher",
              weightedMethod = TRUE,outIntegration = "both",
              ARSmle = "auto",
              ARSdefaultPer = 24,dayZeroBased = FALSE, outSymbol = "",
              parallelize = FALSE,nCores = 1)
            </pre>
          </code>
        <p>
          For each algorithm we obtained different result files for subject 1 and subject 2 and for each subject we
          looked for the significant genes(a significant gene corresponds to a gene for which a circadian rhythm is
          found) from the genes normalized with 10% significance. 42 significant genes were found for JTK (subject1 (0)
          subject2 (42)), for ARS (subject1 (3198) and 7 for subject2), for LS (0 for both subjects).
        </p>
        <p>
          For each gene we have a graph corresponding to the different expression values of this gene over time (6 to
          30 hours), again for this same gene, we get in the meta2d file (which is in one of the files of result meta3d)
          the value of the phase, the base, the period and the amplitude which corresponds to its theoretical curve(this
          is the theoretical curve determined by the algorithm) as a function of time (6 to 30 hours) which is
          superimposed.
        </p>
        <p>Finally for each significant gene found we have a graph with two superimposed curves.<br>For subject 1&#xA0;:</p>

        <div id="slider"> <!-- source : https://lesdocs.fr/slider-javascript/ -->
          <img src="./Data/Metacycle/meta3d_resultats/meta3d_JTK/plot_sujet1/export 1 .png" id="slide">
          <div id="precedent" onclick="ChangeSlide(-1)"><</div>
          <div id="suivant" onclick="ChangeSlide(1)">></div>
        </div>
        <p id="figTitle">Expression of genes as a function of time and according to the JTK method for subject 1.</p>
        
        <p>Then we did the Intersection between the 42 genes detected with JTK and the 3198 genes detected with ARS (Venn diagram).</p>
        <p>
          The Venn diagrams are representations of sets which takes as input a set of data (our files of 42 and 3198
          genes) and as output generates diagrams of the intersection of our data if there are any).
        </p>
        <figure class="venn"><img src="./Data/Metacycle/meta3d_resultats/vendiagram.png" alt="vennDiagram">
          <figcaption>Intersection between the 42 genes detected with JTK and the 3198 genes detected with ARS (Venn
            diagram).</figcaption>
        </figure>
      </article>
      <article>
        <h3>DAVID</h3>
        <p>
          With our significant gene files obtained by JTK (42) and ARS (3198) we went to DAVID to look for an
          enrichment on these genes (at the level of ontology genes) to see if all our genes are mainly associated with
          a set of functions. For that we loaded our gene files in Home Upload Gene List with the other parameters
          (Identifier(OFFICIAL_GENE_SYMBOL), species(Homo sapiens), List Type(Gene List)) then we launch the analysis
          and we obtained a set of results on the enrichment of our genes.
        </p>
      </article>
    </section>
    <section id="conclusion">
      <article>
        <h2>Conclusion, work organization, difficulties encountered, interest and prospects</h2>
        <p>
          On the statistic analysis, the results are low, we have search which genes follow a circadian rhythm in the
          two cancer patients. Thanks to R project, we collected all the statistics we needed to make a conclusion for
          the patients.
        </p>
        <p>We can conclude from this analysis that the two patients have some problems in their clock genes expression.</p>
        <p>
          For the first patient, there are only 42 significant genes with a circadian rhythm, it’s very few in a
          comparison of the number of genes implicated in the circadian rhythm. We find also 3198 genes with an other
          algorithm we saw thanks to DAVID that all these genes find are implicated in similarities functions. They are
          implicated in DNA replication, DNA translation, pH regulation, translation regulation, nucleosome regulation,
          detection of DNA anomaly, extracellular regulations.
        </p>
        <p>
          For the second patient we saw 7 significant genes in all the algorithms. It’s very minimal. These significant
          genes are implicated in the nucleosome regulation and in the extracellular regulation. We lost all the genes
          clocks implicated in the DNA replication and translation, in the pH, the translation, nucleosome regulation.
        </p>
        <p>
          The cancer affects a lot of
          <q><em>clock genes</em></q> expression. In the case of the cell line of B lymphoblastoid cells (patient 1), this
          cancer affects on many genes but there is few functions (only 7 molecular functions) that stay with gene
          having a circadian rhythm.
        </p>
        <p>
          One the other hand, the patient 2 with Hodgkin lymphoma cell line cancer is more affected than the patient 1.
          He has just 7 potentially genes having a circadian rhythm and only 2 molecular functions are stayed with genes
          following a circadian rhythm.
        </p>
        <p>
          In fact, the B lymphoblastoid line cells are affected on the circadian rhythm but it is less affected than
          the Hodgkin lymphoma cell line.
        </p>
        <p>
          We can search more data to make a better analysis including more patients with these types of cancer and also
          more data for the same patient. We could correlate the two researches and see if the first one was significant
          or if it was close to the new one or even if the first one had wrong data in comparison to the second
          research.
        </p>
      </article>
      <article>
        <h3>organization</h3>
        <p>
          We began the project with a big presentation with our referent teacher to introduce the context, which
          software, programs we will use to perform the subject during all the semester. Due to this first conference,
          we decided to make a documentation of our work tools, we choose to work on gitlab and on the nextcloud of our
          university, we implement the KB (student project management) to plan what we need to do in this project and
          who should do what. We have all installed the packages necessary for our statistics analysis on R project. We
          choose to see our referent teacher every week, to say what is done, what is in progress and the eventuals
          questions for the next step of the project.
        </p>
        <p>
          In the beginning of the project, we choose to work alone and to share together what we did. We saw that it
          was not constructive because everyone work the same things with some differences but we thought that it was a
          waste of time. In our project, work on the same thing alone takes more time than work in group conferences,
          when we worked together it was faster, more efficient and we saw a better group cohesion when we decided to
          stop working alone.
        </p>
        <p>
          On the nextcloud, we made a document like a draft report to remember what was done in the first weeks, we
          made it because we had a lot of results on this project and we didn’t want to be lost with all these datas
          made.
        </p>
        <p>
          It was hard to begin the project, our semester was very intense, and it was a bit difficult to be motivated
          into this covid19 context but we persevered and we found our work pace when we began the zoom reunion to work
          together, it was planned, organised, so we progressed.<br> During the semester, we have just used the
          nextcloud of the university, it was sufficient for us and easier to share the work done, we didn’t use the
          gitlab created the first week. We didn’t use the KB, it wasn’t necessary to use it because of the conferences
          each week, we worked step by step and added what we did on the <q><em>draft report</em></q>.
        </p>
      </article>
      <article>
        <h3>Difficulties</h3>
        <p>We had some difficulties during the project, we will explain them from the first one to the last one&#xA0;:</p>
        <ul>
          <li>
            <p>
              We had an enormous data table, with a big part of useless data for our analysis. We needed to make an
              other file to keep only the necessary data. Two of us chose to do that with a python script, it was ok.
              But two have chosen to make it with a AWK script, it was too difficult but with the association of AWK and
              SED, the work can be done. We found two ways to have the file wanted for the rest of the project.
            </p>
          </li>
          <li>
            <p>
              When we began the work on Galaxy, we downloaded all our data necessary for the normalisation, it was ok,
              but when we wanted to make the QC plot, it came out an error. After that, Mister Vandel created an other
              Galaxy instance and all was good after that.
            </p>
          </li>
          <li>
            <p>
              We tried many things on Galaxy and we didn’t really know which parameters we had to put where, so we made
              mistakes, the output files hadn’t the good name of the column or it was the name of the gene which
              appeared and we wanted the username of it. So after these mistakes due to the uncontrol of Galaxy, we made
              a documentation, we made an other file to improve our normalisation. With the good parameters and target
              files, we had the good normalization output files of our data.
            </p>
          </li>
          <li>
            <p>
              When we had our normalization output files of our data, we had analysed it on R project, but an error
              appeared, we didn’t know for a long time why we had this formatting error. When we downloaded the
              normalization output files on R project, the dash became a point. So the table wasn’t a table, it was only
              lines of data. So we transform all the dash in the normalization output files by underscore. We had
              dowload the new file, transform le dash by an underscore was a solution for this problem.
            </p>
          </li>
          <li>
            <p>
              When we began the statistical part on R project, we had two different choices with the package MataCycle,
              we could choose meta2D or meta3D. We refer to the CRAN documentation, we saw that making our analysis with
              meta2D wasn’t possible. In fact, meta2D could make analysis with a large scale periodic data, it was what
              we wanted but, it couldn’t be with meta2D because it made analysis with individual information (we had all
              the information for each patient). We chose to do the meta3D analysis after seeing that it wasn’t possible
              with the meta2D option. To make that, we did the datafile and the designfile with all the necessary
              information.
            </p>
          </li>
        </ul>
      </article>
      <article>
        <h3>Interests and perspectives</h3>
        <h4>Interests&#xA0;:</h4>
        <p>
          There are huge interests for this project, it’s the only time we can make this year a study from the very
          beginning (nothing just a topic), to a conclusion with results. Every time during the year, all our data were
          formatted, and we hadn’t problems with an eventual exchange between a dash and a point. It was interesting to
          take the raw data and make by our own the file we need during the project.
        </p>
        <p>
          It was also the first time for all of us to work on the circadian rhythm, we knew what it was but we never
          thought there were as many genes implicated or correlated with this rhythm. We knew too that cancer disturb
          many things in the human body, but it totally messed the genes implicated in the circadian rhythm, and changed
          or disturb the cellular mechanism of healthy cells.
        </p>
        <p>
          It was interesting to manipulate others logiciels, like AWK combined to SED, or take care that the university
          helped a lot the students with all these applications (the nextcloud, a zoom license, the KB). Thanks to this
          project we had manipulated a lot on R project, it was a good example for the future problems we will have in
          our future jobs.
        </p>
        <p>We have also an interest in our results, we found some genes transformations implicated on a cancer.</p>
        <h4>Perspectives&#xA0;:</h4>
        <p>
          We have some suggestions to be more specific in our study, we think that we should make the same project but
          with more data of other patients with the same pathology of cancer. We can see if these genes transformations
          are due to the pathology of the common cancer or if it’s individual.
        </p>
        <p>
          We think it’s important too to make more data for each patient, if the patient had an other problem during
          the data collection, we could make a false conclusion. But if we do more data, like 24 hours data each week
          during a month for a patient, we could make the average and if something happens just one time, we can
          conclude that it’s not a pathology due to the cancer. We also think to make an other study of the circadian
          rhythm but this time with others biologicals functions, like seeing if the impact of the cancer on the
          circadian rhythm changes things on the organs, on the blood circulation, on the development of the cancer,
          etc...
        </p>
      </article>
    </section>
  </main>
  <footer class="footer">
    <div class="container">
      <div class="text-center">
        <div>Master in Bioinformatics</div>
        <div>Parcours Méthodes Informatiques et Statistiques pour les OMICS</div>
        <div>&copy;<strong> Lidamahasolo Z.</strong> -<strong> Bouanich A.</strong><span> 2021</span></div>
      </div>
    </div>
  </footer>
  
</body>

</html>
